packages/examples/package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "@jscad/examples",
   "version": "2.0.0-alpha.4",
-  "description": "example files for OpenJsCAD",
+  "description": "Example Files for JSCAD",
   "repository": "https://github.com/jscad/OpenJSCAD.org",
   "scripts": {
     "release-patch": "git checkout master && git pull origin master && npm version patch",
     "release-minor": "git checkout master && git pull origin master && npm version minor",
-    "release-major": "git checkout master && git pull origin master && npm version major",
-    "postinstall": "node -e \"console.log('\\u001b[35m\\u001b[1mLove OpenJSCAD? You can now donate to our open collective:\\u001b[22m\\u001b[39m\\n > \\u001b[34mhttps://opencollective.com/openjscad/donate\\u001b[0m')\""
+    "release-major": "git checkout master && git pull origin master && npm version major"
```

Comment:
Won't this get V1?  Use V2 instead of master?

Reply:
Good point. Actually, V2 doesn’t use those NPM commands currently, so I will drop those for now, and remove any kind of doubts.